.travis.yml

```diff
@@ -12,7 +12,10 @@ script:
 - flake8 src
 - cd docs; make html; cd ..;
 - touch ./docs/_build/html/.nojekyll
-- if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then bash ./scripts/validate-version-bump.sh; fi
+# only run the validate-version-bump.sh script on PR builds against master, since those have access to a "master" reference to git show the version
```

Comment:
I actually spot checked this and PRs against other branches appear to have access to a "master" reference as well. Agreed that it's only necessary for PRs against master, though